convert_matrices.py

```diff
@@ -56,13 +56,15 @@ class CSV:
     sep: str = ','
     rows_are_genes: bool = True
     row_filter: Optional[RowFilter] = None
+    kwargs: Optional[dict] = None
 
     def to_mtx(self, input_dir: Path, output_dir: Path):
         converter = CSVConverter(
             input_file=input_dir / self.name,
             delimiter=self.sep,
             rows_are_genes=self.rows_are_genes,
-            row_filter=self.row_filter)
+            row_filter=self.row_filter,
+            **self.kwargs)
```

Comment:
Wouldn't this fail if kwargs is the default value of None? Did you test this for any projects other than the one with the latin1-encoded file?

Reply:
Anyhow, I'm not a fan of the wholesale kwargs pass-though. Please just add an `encoding` field instead.